### PR TITLE
Update top.md

### PR DESCRIPTION
**Pull request by stacykor**
vestigial references to operators when they were a single page. part of PR1379

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

PR1379

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/search/search-query-language/search-operators/top.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ sidebar_label: top
 
 
 
-Use the top operator with the [sort](sort.md) operator, to reduce the number of sorted results returned.
+Use the top operator with the [sort](/docs/search/search-query-language/search-operators/sort/) operator, to reduce the number of sorted results returned.
```
Comment (JainM6) [inline docs/search/search-query-language/search-operators/top.md:10]:
```suggestion
Use the top operator with the [sort](/docs/search/search-query-language/search-operators/sort/) operator to reduce the number of sorted results returned.
```
removed comma

**Review by JainM6 — CHANGES_REQUESTED** [review]
Minor edit

**Review by kimsauce — COMMENTED** [review]
approved w/Megha's suggestion

File: docs/search/search-query-language/search-operators/top.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ sidebar_label: top
 
 
 
-Use the top operator with the [sort](sort.md) operator, to reduce the number of sorted results returned.
+Use the top operator with the [sort](/docs/search/search-query-language/search-operators/sort/) operator to reduce the number of sorted results returned.
```
Comment (kimsauce) [inline docs/search/search-query-language/search-operators/top.md:10]:
```suggestion
Use the top operator with the [sort](/docs/search/search-query-language/search-operators/sort) operator to reduce the number of sorted results returned.
```

File: docs/search/search-query-language/search-operators/top.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ sidebar_label: top
 
 
 
-Use the top operator with the [sort](sort.md) operator, to reduce the number of sorted results returned.
+Use the top operator with the [sort](/docs/search/search-query-language/search-operators/sort) operator to reduce the number of sorted results returned.
 
 :::tip
-We recommend you instead use the [topk](topk.md) operator so you can take advantage of its additional functionality.
+We recommend you instead use the [topk](docs/search/search-query-language/search-operators/topk/) operator so you can take advantage of its additional functionality.
```
Comment (kimsauce) [inline docs/search/search-query-language/search-operators/top.md:13]:
```suggestion
We recommend you instead use the [topk](/docs/search/search-query-language/search-operators/topk) operator so you can take advantage of its additional functionality.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**